ember/app/routes/projects/index.js

```diff
@@ -5,7 +5,10 @@ export default class ProjectsRoute extends Route {
   @service store;
 
   beforeModel() {
-    this.store.unloadAll('project');
+    const unsavedProject = this.store
+      .peekAll('project')
+      .find((project) => !project.status);
```

Comment:
I think there is an `isNew` property for exactly this use case

Reply:
Did you test it afterwards?

Reply:
Implemented here https://github.com/adfinis/Outdated/pull/68/commits/54d10f40f103abdb7c67b02fc6957c6a9981060f

Reply:
tested before and after